### PR TITLE
dom-creation - Susana Gutierrez

### PR DESCRIPTION
**Pull request by susieggrim**
dom-creation

### CODE REVIEW

File: dom-creation/main.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    description: 'It crushes its foe under its heavy body to cause fainting. In a pinch, it will withdraw inside its shell.',
+    imageUrl: 'images/blastoise.png'
+  }
+];
+
+function renderPokemon(pokemon) {
+
+  var elementDiv = document.createElement('div');
+  elementDiv.setAttribute('class', 'column-third');
+  document.querySelector('.row').appendChild(elementDiv);
```
Comment (BrettAlbright) [inline dom-creation/main.js]: so the one thing to update is to move the appending to the DOM to outside of the function. The `renderPokemon` function should only create and return the DOM tree, not append it to the DOM. 

You want to append each completed DOM tree to the DOM within the loop.

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
small update needed!

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Comment by susieggrim** [comment]
Changes were completed.
dom-creation-2

**Pull request merged**